core/src/Slay/Prim.hs

```diff
@@ -82,6 +86,18 @@ data PrimCurve g =
       curveDirection :: g Direction
     }
 
+
+-- | circle adt
+data PrimCircle g = Circle
+    { circleColor :: g (Maybe Color)
+    , circleCenterX :: Double
```

Comment:
I removed them so we can use Offset values in the RenderElement instance. What do we do if someone specifies radius that is larger than offset ?